examples/04_pong/audio.rs

```diff
@@ -60,10 +61,12 @@ pub fn initialise_audio(world: &mut World) {
 }
 
 /// Plays the bounce sound when a ball hits a side or a paddle.
-pub fn play_bounce(sounds: &Sounds, audio_output: &Option<Output>) {
+pub fn play_bounce(sounds: &Sounds, storage: &AssetStorage<Source>, output: &Option<Output>) {
     use amethyst::audio::play::play_once;
 
-    if let Some(ref audio_output) = *audio_output {
-        play_once(&sounds.bounce_sfx, 1.0, &audio_output);
+    if let Some(ref output) = output.as_ref() {
+        if let Some(sound) = storage.get(&sounds.bounce_sfx) {
```

Comment:
Maybe we could add a function to the AssetStorage that assumes the handle is valid and returns a non-option type, in the event that it's not valid just panic.  Might help reduce verbosity in situations like this where we're reasonably confident our handle is correct.

Reply:
We can, but it would be unsafe, thus not a measure for reducing verbosity, but overhead.